script/Arcueid/Day 8/08_00_ARC08_8_2.txt

```diff
@@ -48,7 +48,7 @@ Two steps. Three steps. Four steps.
 [sha:aa8203aa52b28e5cc413594710592998df9f8c3e]{
 -- Page 584, Offset 21250. Glued.
 -- ―――あと一歩。
-―――One more step to go.
+ ―――One more step to go.
```

Comment:
Did you check this in game? I don't recall needing the space.

Reply:
Might then be an issue with my game, since I only update the script mrg

Reply:
Ah, yeah it is glued on the same line. I'd honestly be tempted to stick it on a newline (put a `%{n}` on the front)